### PR TITLE
Property configurator (ios)

### PR DESCRIPTION
**Pull request by annakocheshkova**
iOS implementation of `PropertyConfigurator`.

### CODE REVIEW

**Review by max-mironov — DISMISSED** [review]

File: Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+#import <Foundation/Foundation.h>
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.h]: Redundant

File: Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+@class MSPropertyConfigurator;
+extern "C" void appcenter_unity_property_configurator_set_app_name(MSPropertyConfigurator *configurator, char* appName);
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.h]: Add empty line before

File: Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.mm [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+#import "WrapperPropertyConfigurator.h"
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Analytics/PropertyConfigurator.mm]: Wrapper? Please make sure that it builds

File: Assets/AppCenter/Plugins/iOS/Analytics/TransmissionTarget.h [hunk 4]
```diff
@@ -5,8 +5,10 @@
 #import <Foundation/Foundation.h>
 
 @class MSAnalyticsTransmissionTarget;
+@class MSPropertyConfigurator;
 extern "C" void appcenter_unity_transmission_target_track_event(MSAnalyticsTransmissionTarget *transmission, char* eventName);
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Analytics/TransmissionTarget.h]: Empty line before

File: Assets/AppCenter/Plugins/iOS/Analytics/TransmissionTarget.mm [hunk 5]
```diff
@@ -4,6 +4,7 @@
 
 #import "WrapperTransmissionTarget.h"
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Analytics/TransmissionTarget.mm]: Wrapper?

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**